### PR TITLE
Add pre-/postsubmit api-gateway integration tests (build job guarded)

### PR DESCRIPTION
**Pull request by videlov**
**Description**

Changes proposed in this pull request:

- Adding postsubmit api-gateway integration test
- Job guard for the api-gateway integration tests

**Related issue(s)**

### CODE REVIEW

File: vpath/pjtester.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+pjConfigs:
```
Comment (Sawthis) [inline vpath/pjtester.yaml:1]: Please remove this file.

**Review by Sawthis — CHANGES_REQUESTED** [review]

**Review by Ressetkk — APPROVED** [review]
keep in mind jobguard is not a recommended solution, and ProwJobs should not be generally chained this way

LGTM otherwise.

**Pull request merged**